### DISCUSSION TITLE
Crazy visual Bug

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#1210] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. NC AIO : latest stable
2. Open The Files: Chrome Browser (mobile)
3. scroll through files and folders

### Expected behavior 
List of folders and files

### Actual behavior 
super crazy confusing visual bugs (as if the graphics module was broken)

### Host OS 
Ubuntu 22.04 lts

#### Nextcloud AIO version 
Latest

#### Current channel 
Stable

#### Other valuable info 
![Screenshot_2022-09-30-12-39-33-71_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://user-images.githubusercontent.com/90777102/193252695-364ee371-6d18-4b3e-9af1-551a5e11dd99.jpg)

**Comment by szaimen** [comment, marked as answer]
Hm... looks like a browser issue or graphics card issue.
